packages/adminrouter/docker/adminrouter-upstreams-open.conf

```diff
@@ -1,17 +1,14 @@
 set $upstream_networking_api http://127.0.0.1:61430;
 
-proxy_ssl_trusted_certificate /run/dcos/pki/CA/certs/ca.crt;
+proxy_ssl_trusted_certificate /run/dcos/pki/CA/ca-bundle.crt;
```

Comment:
The problem here is that due to pkgpanda limitations there is no easy way to keep docker files DRY :(